hack/run-all.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Run all the important scripts in a sensible order,
+# handy for switching branches, etc
+wd="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
Ben's approach gives you the absolute path to the directory, Sam's (what we use in Origin) is a relative path. Shouldn't make any difference, really. If you're keeping this approach, keep your quotes the way they are in case `$ BASH_SOURCE` contains spaces.